### PR TITLE
Added unit test folder structure and unit test for beer styles

### PR DESCRIPTION
**Pull request by edjroz**
added tests folder to store unit tests and worked on issue #54

### CODE REVIEW

**Comment by MichaelWashburnJr** [comment]
Awesome! I'll try to review tonight between flights or early tomorrow. Thanks!

File: brewcenter_api/brew_data/tests/test_style_model.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+from django.test import TestCase
+from ..models.style import Style
+
+
+class StyleTest(TestCase):
```
Comment (MichaelWashburnJr) [inline brewcenter_api/brew_data/tests/test_style_model.py]: A good test should setup and destroy the data it uses, leaving no trace. Think you could make a tearDown method?

Comment (edjroz) [reply brewcenter_api/brew_data/tests/test_style_model.py]: Will do, is the folder structure ok? for the project or should it be added to the tests.py, I think its better with a test directory to minimize confusion

Comment (MichaelWashburnJr) [reply brewcenter_api/brew_data/tests/test_style_model.py]: I prefer folder structures rather than a single file.

**Pull request closed without merging**